src/main/scala/beam/agentsim/agents/Population.scala

```diff
@@ -33,21 +33,20 @@ class Population(val scenario: Scenario, val beamServices: BeamServices, val sch
     case _: Exception => Stop
     case _: AssertionError => Stop
   }
-  private implicit val timeout = Timeout(50000, TimeUnit.SECONDS)
+  private implicit val timeout: Timeout = Timeout(50000, TimeUnit.SECONDS)
 
   import context.dispatcher
 
-  private var personToHouseholdId: Map[Id[Person], Id[Household]] = Map()
+  private val personToHouseholdId: mutable.Map[Id[Person], Id[Household]] = mutable.Map[Id[Person], Id[Household]]()
```

Comment:
As this is only done at creation then it seems the only reason mutability was used was because of the initialization. So, I'd argue for something more like this:

```
  private val personToHouseholdId: scala.collection.Map[Id[Person], Id[Household]] = 
    scenario.getHouseholds.getHouseholds.asScala
      .flatMap{ case (householdId, matSimHousehold) =>
        matSimHousehold.getMemberIds.asScala.map(_ -> householdId)
      }
```

Of course if you feel the performance of mutation to be better, then you could do something more like this and end with the same result:

```
  def initPersonToHouseholdId(houseHolds: java.util.Map[Id[Household], Household]): scala.collection.Map[Id[Person], Id[Household]] = {
    val personToHouseholdId: mutable.Map[Id[Person], Id[Household]] = mutable.Map[Id[Person], Id[Household]]()
    scenario.getHouseholds.getHouseholds.forEach { (householdId, matSimHousehold) =>
      personToHouseholdId ++= matSimHousehold.getMemberIds.asScala.map(personId => personId -> householdId)
    }
    personToHouseholdId.toMap
  }
  
  private val personToHouseholdId: scala.collection.Map[Id[Person], Id[Household]] = 
    initPersonToHouseholdId(scenario.getHouseholds.getHouseholds)
```